### PR TITLE
Implement /upload mapping in Controller

### PR DESCRIPTION
**Pull request by alszla**
Now a file can be uploaded through the "/upload" endpoint.
There's also possible to download a file through the "/download" endpoint.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/controller/StorageService.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Implements a Spring service that handles copying to the upload directory.
+ *
+ * @author Alejandro S�nchez Luizaga
+ * @version 1.0
+ */
+@Service
+public class StorageService {
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/controller/StorageService.java]: Is it necessary this space?

File: src/main/java/com/fundation/webservice/controller/StorageService.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    private final Path uploadLocation;
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/controller/StorageService.java:34]: Is it necessary this space?

File: src/main/java/com/fundation/webservice/controller/StorageService.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    @Autowired
+    public StorageService(StorageProperties storageProperties) {
+        this.uploadLocation = Paths.get(storageProperties.getUploadDir()).toAbsolutePath()
+                              .normalize();
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/controller/StorageService.java]: Is it necessary this space?

File: src/main/java/com/fundation/webservice/controller/VideoResponse.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.controller;
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/controller/VideoResponse.java:11]: Is it necessary this space?

File: src/main/java/com/fundation/webservice/controller/StorageService.java (hunk 1 above)
Comment (alszla) [reply src/main/java/com/fundation/webservice/controller/StorageService.java]: (Line 33) Corrected.

File: src/main/java/com/fundation/webservice/controller/StorageService.java (hunk 2 above)
Comment (alszla) [reply src/main/java/com/fundation/webservice/controller/StorageService.java:34]: Yes. This (Line 35) is a space placed between attribute and method.

File: src/main/java/com/fundation/webservice/controller/StorageService.java (hunk 3 above)
Comment (alszla) [reply src/main/java/com/fundation/webservice/controller/StorageService.java]: Corrected.

File: src/main/java/com/fundation/webservice/controller/VideoResponse.java (hunk 4 above)
Comment (alszla) [reply src/main/java/com/fundation/webservice/controller/VideoResponse.java:11]: Yes. Actually, this space separates the package section from the import section.

File: src/main/java/com/fundation/webservice/controller/StorageService.java (hunk 2 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/controller/StorageService.java:34]: ok my friend, that's good.

File: src/main/java/com/fundation/webservice/controller/StorageService.java (hunk 3 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/controller/StorageService.java]: ok

File: src/main/java/com/fundation/webservice/controller/VideoResponse.java (hunk 4 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/controller/VideoResponse.java:11]: ok

**Comment by JosueRodriguezGarcia** [comment]
ok

File: src/main/java/com/fundation/webservice/controller/StorageService.java (hunk 1 above)
Comment (alszla) [reply src/main/java/com/fundation/webservice/controller/StorageService.java]: The extra line has now been removed. Somehow I had forgotten to push the changes. Sorry

**Pull request merged**